File: Assignment8/C/Program3.c
// PS - Write a program which acepts one number from user and return count of digits in between 3 and 7

#include<stdio.h>

int CountInbetween(int iNo) {
    int iDigit = 0;
    int iCount = 0;

    if(iNo < 0) {
        iNo = -iNo;
    }

    while(iNo != 0) {
        iDigit = iNo % 10;
        if((iDigit > 3) && (iDigit < 7)){
            iCount++;
        }
        iNo = iNo / 10;
    }
    return iCount;
}

int main() {
    int iValue = 0;
    int iRet = 0;

    printf("Enter number \n");
    scanf("%d", &iValue);

    iRet = CountInbetween(iValue);

    printf("Count of digits in between 3 and 7 is %d \n", iRet);

    return 0;
}

File: Assignment35/Java/Program3.java
// PS - Write a java program which accept number from user and return count of digits are in between 3 and 7

import java.util.*;

class DigitDemo
{
    public int CountRange(int No)
    {
        int range = 0;
        int iDigit = 0;

        while(No != 0)
        {
            iDigit = No % 10;
            if((iDigit >= 3) && (iDigit <= 7))
            {
                range++;
            }
            No = No / 10;
        }
        return range;
    }
}

class Program3
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int No = sobj.nextInt();

        DigitDemo dobj = new DigitDemo();
        int iRet = dobj.CountRange(No);

        System.out.println("Count of digits are : "+iRet);
    }
}